### ISSUE TITLE
And what about Android accessibility suite support

### ISSUE THREAD

**Issue by JanuszChmiel** [cnlohr/rawdrawandroid#38]
Do you think, that there could be some complex C language routine, which would allow developers to create fully accessible GUi even for Android accessibility suite users? Or unfortunately, GUI compatible with Explore by touch Android service and with Android accessibility suite can not be done by this way?

You are C language expert, so I AM giving you this question.

**Comment by dreua** [comment]
I'm not sure I understand your question correctly, could you give me a use case or some information of the motivation behind this question? As I see it, rawdrawandroid is more or less a proof of concept and the only published app using it, colorchord, does not have any user controls that could pose accessibility problems.

**Comment by SelfTide** [comment]
I think he is talking about a port of GUI like Qt.... but, maybe say GTK?

**Comment by dreua** [comment]
That's not what rawdraw android is going to do. Btw, you can already build Qt apps for Android, it's bulky and heavy but it works, if you out the effort in. For Gtk I'm not sure but that would be another project anyway.

I'm closing this as there has been no response from the requestor.
